### PR TITLE
TasksController priorities

### PR DESCRIPTION
**Pull request by alexandraorth**

### CODE REVIEW

File: grakn-engine/src/main/java/ai/grakn/engine/controller/TasksController.java [hunk 1]
```diff
@@ -100,21 +95,21 @@ public TasksController(Service spark, TaskManager manager) {
     })
     private Json getTasks(Request request, Response response) {
         TaskStatus status = null;
-        String className = request.queryParams(TASK_CLASS_NAME_PARAMETER);
-        String creator = request.queryParams(TASK_CREATOR_PARAMETER);
+        String className = request.queryParams(REST.Request.TASK_CLASS_NAME_PARAMETER);
```
Comment (fppt) [inline grakn-engine/src/main/java/ai/grakn/engine/controller/TasksController.java]: YESSSSS!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! I hate static imports of strings and enums.

File: grakn-engine/src/main/java/ai/grakn/engine/controller/TasksController.java [hunk 2]
```diff
@@ -179,6 +177,9 @@ private Json createTask(Request request, Response response) {
                     .map(interval -> recurring(time, interval))
                     .orElse(TaskSchedule.at(time));
 
+            // Get the priority of a task (default is low)
+            priority = Optional.ofNullable(priorityParam).map(TaskState.Priority::valueOf).orElse(DEFAULT_TASK_PRIORITY);
```
Comment (fppt) [inline grakn-engine/src/main/java/ai/grakn/engine/controller/TasksController.java]: Nitpick: What happens if I send the priority "asdfkhjvd". I expect a nasty exception?

Comment (alexandraorth) [inline grakn-engine/src/main/java/ai/grakn/engine/controller/TasksController.java]: I have a test for this! You get back an IllegalArgumentException and a server 400.

Comment (alexandraorth) [inline grakn-engine/src/main/java/ai/grakn/engine/controller/TasksController.java]: Which might not be the desired behavior :P

Comment (fppt) [inline grakn-engine/src/main/java/ai/grakn/engine/controller/TasksController.java]: Ooooo nice.

**Pull request merged**